specs/language/statements.tex

```diff
@@ -1,2 +1,93 @@
 \Ch{Statements}{Stmt}
```

Comment:
Can we add a grammar definition up here? Something like:

```
\begin{grammar}
  \define{statement}\br
  labeled-statement\br
  \opt{attribute-specifier-sequence} expression-statement\br
  \opt{attribute-specifier-sequence} compound-statement\br
  \opt{attribute-specifier-sequence} iteration-statement\br
  \opt{attribute-specifier-sequence} selection-statement\br
  declaration-statement
\end{grammar}
```

Reply:
It Looked weird to me to not have a section header for the grammar. If thats ugly to you I can delete it.

Reply:
Yea, I'd prefer no section for it. We haven't done sections for grammar entries in other places.